api/api.go

```diff
@@ -2,26 +2,31 @@ package api
 
 import (
 	"context"
-
-	store "github.com/ONSdigital/dp-search-reindex-api/store"
+	"github.com/ONSdigital/log.go/log"
 	"github.com/gorilla/mux"
 )
 
 //JobStoreAPI provides a struct to wrap the api around
 type JobStoreAPI struct {
-	Router   *mux.Router
-	jobStore store.JobStore
+	Router  *mux.Router
+	mongoDB JobStorer
```

Comment:
I'd still call this field `jobStore` as the `api` package doesn't really care where the job is stored to, it just wants to store the job somewhere.

Reply:
makes sense - done